### ISSUE TITLE
Font resizing in reader mode

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero-ios#685]
https://forums.zotero.org/discussion/comment/434461/#Comment_434461

**Comment by michalrentka** [comment]
@dstillman the reader mode in our PDF reader supports accessibility, so if they increate font size in iOS settings, the text is bigger:
![IMG_E3AB258905EA-1](https://github.com/zotero/zotero-ios/assets/1022956/968bf183-1103-4800-b27a-72efa83c57e2)

**Comment by dstillman** [comment]
Yeah, but it’s pretty standard to have font size controls in a reader mode anyway (e.g., Safari on iOS).

**Comment by michalrentka** [comment]
I tried to make it work, but it seems the PSPDFKit component doesn't support it. I contacted their support to see whether I missed something or they can implement it later at least.

**Comment by michalrentka** [comment]
I got response from PSPDFKit, not currently supported. They created a ticket and will hopefully add this sometime.
